tests/src/test/java/alluxio/hadoop/fs/DFSIOIntegrationTest.java

```diff
@@ -176,6 +176,9 @@ static ByteMultiple parseString(String sMultiple) {
     }
   }
 
+  /**
+   * Init DFSIOIntegrationTest.
```

Comment:
I don't think this comment is necessary, we only javadoc tests when they would be confusing otherwise